crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -273,44 +255,26 @@ impl ViewEye {
         // Temporarily switch to orbital, set the values, and then switch back.
         // This ensures the camera position will be set correctly, even if we
         // were in first-person mode:
-        let old_mode = self.mode();
-        self.set_mode(EyeMode::Orbital);
+        let old_mode = self.kind();
+        self.set_kind(Eye3DKind::Orbital);
         self.center = orbit_center;
         self.orbit_radius = orbit_radius;
-        self.set_mode(old_mode);
+        self.set_kind(old_mode);
     }
 
     /// The world-space position of the eye.
     pub fn position(&self) -> Vec3 {
-        match self.mode {
-            EyeMode::FirstPerson => self.center,
-            EyeMode::Orbital => self.center - self.orbit_radius * self.fwd(),
+        match self.kind {
+            Eye3DKind::FirstPerson => self.center,
+            Eye3DKind::Orbital => self.center - self.orbit_radius * self.fwd(),
         }
     }
 
-    /// Compute the actual speed depending on the [`EyeMode`].
-    fn fallback_speed_for_mode(&self, bounding_boxes: &SceneBoundingBoxes) -> f32 {
-        match self.mode {
-            EyeMode::FirstPerson => 0.1 * bounding_boxes.current.size().length(),
-            EyeMode::Orbital => self.orbit_radius,
-        }
-    }
-
-    /// Returns the actual speed (float) of [`ViewEye`].
-    pub fn speed(&self, bounding_boxes: &SceneBoundingBoxes) -> f32 {
-        match self.speed {
-            CameraTranslationSpeed::Auto => self.fallback_speed_for_mode(bounding_boxes),
-            CameraTranslationSpeed::Override(speed) => speed,
-        }
-    }
-
-    /// Set the speed to a specific value set by the user via the UI.
-    ///
-    /// `None` resets speed to the default.
-    pub fn set_speed(&mut self, new_speed: Option<f32>) {
-        match new_speed {
-            Some(speed) => self.speed = CameraTranslationSpeed::Override(speed),
-            None => self.speed = CameraTranslationSpeed::Auto,
+    /// Compute the actual speed depending on the [`Eye3DKind`].
+    pub fn fallback_speed_for_mode(&self, bounding_boxes: &SceneBoundingBoxes) -> f32 {
```

Comment:
```suggestion
    pub fn fallback_speed_for_kind(&self, bounding_boxes: &SceneBoundingBoxes) -> f32 {
```

let's be consequent :)